src/pkg/runner/actions.go

```diff
@@ -26,6 +26,24 @@ import (
 	zarfTypes "github.com/defenseunicorns/zarf/src/types"
 )
 
+// Utils is an interface that defines the GetFinalExecutablePath method.
+type Utils interface {
```

Comment:
A suggestion for the name of the interface`ExecutablePathFinder`

In `go`, it is standard to use `er` as interface `suffix` https://stackoverflow.com/questions/54213897/rule-for-golang-interface-name-and-its-number-of-methods

Reply:
addressed in latest commit